ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -651,34 +680,38 @@ abstract class AbstractHttpServiceIntegrationTest
   "multi-party query GET" in withHttpService { (uri, encoder, _, _) =>
     for {
       _ <- postCreateCommand(
-        accountCreateCommand(owner = domain.Party("Alice"), number = "42"),
+        accountCreateCommand(owner = domain.Party("Carol"), number = "42"),
```

Comment:
These are very common names that could lend themselves to conflicts. If we want to re-use the same ledger over and over again we should probably also create a mechanism to create party names on a per-test basis (e.g. adding a prefix or a suffix to each party name) to avoid naming conflicts that could cause some unnecessary head scratch in the process of adding further tests in the future.

Reply:
That's a valid concern, I'll think about a way to do this but at the same time I am open to any easy solutions/suggestions in this regard.